bin/spawn_devcontainer.sh

```diff
@@ -107,29 +107,11 @@ NVIM_DEVCONTAINER_CLI_FOLDER_IN_DOCKER_CONTAINER=${REMOTE_HOME}"${NVIM_DEVCONTAI
 DOTFILES_DIR="${REMOTE_HOME}/$nvim_dotfiles_dir"
 SETUP_ENVIRONMENT_DIR="${REMOTE_HOME}/$setup_environment_dir"
 
-# Check if file .config/github-copilot exists
-if [ ! -d "${HOME}"/.config/github-copilot ]; then
-	echo "File ${HOME}/.config/github-copilot does not exist"
-else
-	MOUNT_BIND_COPILOT="--mount type=bind,source=${HOME}/.config/github-copilot,target=${REMOTE_HOME}/.config/github-copilot"
-fi
-
-# Check if file .gitconfig exists
-if [ ! -f "${HOME}"/.gitconfig ]; then
-	echo "File ${HOME}/.gitconfig does not exist"
-else
-	MOUNT_BIND_GITCONFIG="--mount type=bind,source=${HOME}/.gitconfig,target=${REMOTE_HOME}/.gitconfig"
-fi
-
-
 # Mount the plugin folder only if --env is set to dev:
-if [ "$env" = "dev" ]; then
-	MOUNT_BIND_PLUGIN="--mount type=bind,source=${HOME}/${NVIM_DEVCONTAINER_CLI_FOLDER},target=${NVIM_DEVCONTAINER_CLI_FOLDER_IN_DOCKER_CONTAINER}"
-fi
+# if [ "$env" = "dev" ]; then
+# 	MOUNT_BIND_PLUGIN="--mount type=bind,source=${HOME}/${NVIM_DEVCONTAINER_CLI_FOLDER},target=${NVIM_DEVCONTAINER_CLI_FOLDER_IN_DOCKER_CONTAINER}"
```

Comment:
Why is this mount needed? I just hadn't seen why it is necessary.

Reply:
This was only necessary when I'm developing the current plugin inside a devcontainer (that's when the `dev` mode was activated). However, I would say that we can do the same thing by having this mount in the devcontainer.json instead of having this config hardcoded in the plugin.

Reply:
I for some reason didn't notice it was commented out. Why not just delete it? It will be in the git history if you really need it. But otherwise, yeah, I think that doing everything via the devcontainer is the way to go.

Reply:
Just removed from the code!